packages/playground/devEnv/build.ts

```diff
@@ -51,17 +55,45 @@ export async function start(options: {
           return [
             path.basename(groupDir),
             Object.fromEntries(
-              readdirSync(groupDir).map((moduleDirName) => [
-                path.basename(moduleDirName),
-                {
-                  entryDir: path.join(groupDir, moduleDirName),
-                  outDir: path.join(
-                    buildDir,
-                    path.basename(groupDir),
-                    moduleDirName,
-                  ),
-                },
-              ]),
+              readdirSync(groupDir)
```

Comment:
This is still for internal. I wrote this before really considering the split between end-user building and internal building. Since it was already written (and it's fairly straightforward and it could reduce friction for internal dev), I opted to keep it.

When picking an end-user tool for playing with Theatre, I'd certainly make sure this just happens automatically via the bundler we choose.